page-impact.hbs

```diff
@@ -0,0 +1,550 @@
+{{!< default}}
+
+<div class="gh-page">
+    <div class="gh-container">
+
+        <!-- IMPACT-OUR-IMPACT -->
+        <div class="row">
+            <div class="col-xs-12 col-md-12 col-lg-12 mb-5">
+                <h1>Our Impact</h1>
+            </div>
+            <div class="col-xs-12 col-md-8 col-lg-8">
+                <article class="gh-card {{post_class}} {{#if feature_image}}with-image{{else}}no-image{{/if}}">
```

Comment:
For L12-39, I think this shouldn't use the `article` tag or any of the attributes associated with posts. Could this be changed to a `div` instead with `{{post_class}}` and `{{#if feature_image}}...{{/if}}` removed?